WcaOnRails/app/controllers/api/v0/persons_controller.rb

```diff
@@ -37,6 +37,10 @@ def results
       medals: person.medals,
       records: person.records,
     }
+    if show_competitions == true
+      person_json["competitions"] = person.competitions
```

Comment:
I think our convention is to populate the result json by `:symbol`, not by `"string"`. See `api_controller` for samples.